scrape.js

```diff
@@ -172,13 +177,31 @@ const scrapeLinkedIn = async (data) => {
 
       console.log(`Active users on page ${pageNumber}: `, activeEmployees);
 
+      if (!fs.existsSync("./output")) {
+        // check for existing output directory, create it if necessary
+        fs.mkdirSync("./output");
+      }
+
+      const pageName = `page${pageNumber}`;
+      let currPage = [];
+      activeEmployees.forEach((employee) => {
+        currPage.push(employee);
+      });
+      pages.push({ pageName: currPage });
```

Comment:
Sorry to block this PR again. There's a small bug here. When you run the script, you get `pageName` instead of `page0`, `page1`, etc. in the output file. This is because when you push `{pageName: currPage}` into a JSON, it uses `pageName` as a string instead of a variable. One way to solve this is to replace `pageName` with `[pageName]`.

Reply:
Whoops, can't believe I glanced over that! Fixing now

Reply:
Should be all set now :) let me know if there's anything else!